packages/analytics-browser/generated/amplitude-gtm-snippet.js

```diff
@@ -0,0 +1,96 @@
+"use strict";
+
+/**
+ * Imported in client browser via <script> tag
+ * Async capabilities: Interally creates stubbed window.amplitudeGTM object until real SDK loaded
+ * Stubbed functions keep track of funciton calls and their arguments
+ * These are sent once real SDK loaded through another <script> tag
+ */
+!function (window, document) {
+  var amplitude = window.amplitudeGTM || {
+    _q: [],
+    _iq: {}
+  };
+  if (amplitude.invoked) window.console && console.error && console.error('Amplitude snippet has been loaded.');else {
+    var proxy = function proxy(obj, fn) {
+      obj.prototype[fn] = function () {
+        this._q.push({
+          name: fn,
+          args: Array.prototype.slice.call(arguments, 0)
+        });
+        return this;
+      };
+    };
+    var getPromiseResult = function getPromiseResult(instance, fn, args) {
+      return function (resolve) {
+        instance._q.push({
+          name: fn,
+          args: Array.prototype.slice.call(args, 0),
+          resolve: resolve
+        });
+      };
+    };
+    var proxyMain = function proxyMain(instance, fn, isPromise) {
+      var args = arguments;
+      instance[fn] = function () {
+        if (isPromise) return {
+          promise: new Promise(getPromiseResult(instance, fn, Array.prototype.slice.call(arguments)))
+        };
+      };
+    };
+    var setUpProxy = function setUpProxy(instance) {
+      for (var k = 0; k < funcs.length; k++) {
+        proxyMain(instance, funcs[k], false);
+      }
+      for (var l = 0; l < funcsWithPromise.length; l++) {
+        proxyMain(instance, funcsWithPromise[l], true);
+      }
+    };
+    amplitude.invoked = true;
+    var as = document.createElement('script');
+    as.type = 'text/javascript';
+    as.integrity = 'sha384-slx5+gMeTDnjn4NnEcw9OwBVFUI6M4Od3bVcWwMw6owXGvsJOla40Vg5gNCXxNvq';
+    as.crossOrigin = 'anonymous';
+    as.async = true;
+    as.src = 'https://cdn.amplitude.com/libs/analytics-browser-gtm-2.1.2-min.js.gz';
```

Comment:
Curious how do we get the integrity and src?

Reply:
https://github.com/amplitude/Amplitude-TypeScript/blob/main/scripts/version/create-snippet.js#L34 and https://github.com/amplitude/Amplitude-TypeScript/blob/main/scripts/templates/browser-snippet.template.js#L12